var/spack/repos/builtin/packages/uftrace/package.py

```diff
@@ -37,6 +38,12 @@ class Uftrace(AutotoolsPackage):
     depends_on('python@3.5:', when='+python3')
     depends_on('lua-luajit')
 
+    # Fix the version string if building below another git repo. Submitted to upstream:
+    when('@:0.11')
```

Comment:
Did you mean to add a decorator?

Reply:
flake8 required me to add the empty line. It's ok, no problem, I wouldn't want to start a new pipeline for that.

Reply:
@bernhardkaindl I think this should be:
```
@when('@:0.11')
def patch(self):
    ...
```
if the intent is to have the function run only on versions not greater than 0.11